pandas/tests/io/json/test_pandas.py

```diff
@@ -1647,3 +1647,25 @@ def test_tuple_labels(self, orient, expected):
         df = pd.DataFrame([[1]], index=[("a", "b")], columns=[("c", "d")])
         result = df.to_json(orient=orient)
         assert result == expected
+
+    @pytest.mark.parametrize("indent", [1, 2, 4])
+    def test_to_json_indent(self, indent):
```

Comment:
i would add indent for some existing tests as this doesn’t test all of the code paths 

eg for different types i don’t think u r fully testing 

pick a value like 4 and use it consistently

Reply:
Sounds good. I'm thinking adding something for each orient will suffice, though if you are thinking more beyond that lmk

Reply:
looks good.